README.md

```diff
@@ -6,4 +6,39 @@
 [travis]: https://travis-ci.org/Ensembl/ensembl
 [coveralls]: https://coveralls.io/github/Ensembl/ensembl
 
+The Ensembl Core API (Application Programme Interface) serves as a middle layer between the underlying MySQL database and the user's script. It aims to encapsulate the database layout by providing high level access to the database.
 
+Find more information (including the installation guide and a tutorial) on the Ensembl website: http://www.ensembl.org/info/docs/api/core/
+
+
+## Download
+To clone the Ensembl Core API, use the following command:
+
+```
+git clone https://github.com/Ensembl/ensembl.git
+```
+
```

Comment:
Alternatively, you can download the files in gzipped TAR format from our [ftp site](ftp://ftp.ensembl.org/pub/ensembl-api.tar.gz)